README.md

```diff
@@ -1,27 +1,87 @@
+|---------------|-----------------|-------------------------------------------------|-------------------------------------------------------------------------------------------|:-------------------------------:|
+| `name`        | string          |                                                 | Name of the interface to configure, ignored if `mac` is specified                         |  only if `mac` is not specified |
+| `mac`         | string          |                                                 | MAC-Address of the interface to configure. If `name` is specified, `mac` is autodetected. | only if `name` is not specified |
+| `bring_up`    | boolean         | `{{ network_management_default_bring_up }}`     | Automatically bring interface link up                                                     |                N                |
+| `type`        | [string](#type) | `{{ network_management_default_type }}`         | Specify how the network interface should be configured                                    |                N                |
+| `dhcp`        | [dict](#dhcp)   | `{{ network_management_default_dhcp_options }}` | Additional options for dhcp interfaces                                                    |                N                |
+| `nameservers` | list of strings | `{{ network_management_nameservers }}`          | Nameservers to use                                                                        |                N                |
+| `mtu`         | integer         |                                                 | MTU size                                                                                  |                N                |
+| `ip`          | string          |                                                 | IP address to assign as CIDR                                                              |                N                |
+| `ips`         | list of strings |                                                 | IP addresses to assign to interface as CIDR                                               |    only if `type` is `static`   |
+| `gateway`     | string          | `{{ network_management_default_gateway }}`      | Add a default route via this gateway address                                              |                N                |
+
+
+### bridges
+Beside every option from the [interfaces](#interfaces) dict, the following options can be specified:
+
+| Option  | Type                   | Default | Description                                      | Required |
+|---------|------------------------|---------|--------------------------------------------------|:--------:|
+| `ports` | [list of dicts](#port) | `[]`    | List of physical interfaces to add to the bridge |     N    |
+
+
+## ToDo
```

Comment:
Please move these to enhancement or feature issues

Reply:
fixed